bin/dependency-check-pr.sh

```diff
@@ -2,6 +2,9 @@
 
 set -eou pipefail
 IFS=$'\n\t'
+white="\e[97m"
+green="\e[32m"
+reset="\e[0m"
```

Comment:
seeing these defined in multiple files. Would love to see a helper script that can be sourced with the values defined within functions— so something like

```
#!/bin/bash
# no flags here

white="\e[97m"
green="\e[32m"
reset="\e[0m"

echo_green(){
  echo -e "Current Tag: ${green}${1:-}${reset}"
}
```
then in this script you `source $INC/echo-colors.sh` and then echo_white "foo"

Reply:
hmm...not sure I'm kosher with the idea of helper functions. Because it's used in different contexts, actual usage would need to be something like `echo -e "Current Tag: $(echo_green $(1:-))"` because that's not the only text that we would output -- and at that point there's not much difference between that and the current syntax. However sourcing the color variables might be worth doing. Not sure it's worth it for this PR though, since it's just cosmetic.

Reply:
ah you're right I wasn't capturing the fact that part of the string is not green. agree sourcing the colors separately can be done elsewhere.